util/make_timeline.py

```diff
@@ -97,14 +139,29 @@ def parse_file(args):
 
     entries = []
     started = False
-
+    encounter_sets = []
     with args.file as file:
+        # If searching for encounters, divert and find start/end first3
```

Comment:
typo?

Reply:
Yes. Not sure why I would have been using the number pad for that, but 3 is directly above the period key there.